.drone.yml

```diff
@@ -42,43 +43,38 @@ pipeline:
 
   # pr
 
-  sbt-update:
-    <<: *sbtenv
-    commands:
-      - sbt update
-
   compile:
     <<: *sbtenv
     commands:
-      - sbt rholang/bnfc:generate compile test:compile it:compile doc
-
-  run-unit-tests:
-    <<: *sbtenv
-    commands:
-      - sbt test it:test
-
-  run-code-analysis:
-    <<: *sbtenv
-    commands:
-      - sbt scalafmtCheck coverage
-      - ./scripts/ci/codecov-upload
```

Comment:
So we no longer run code coverage? I'm not against it if so, I'm just wondering.

Reply:
See the commit message and/or discord. We haven't been running it for 2 months already, due to some kind of bug/misconfig. It's out of this PRs scope to fix that. This change is merely a clean up making the status quo explicit :)

Reply:
I like how descriptive your commit messages are.